src/Dialog/Confirmation/ConfirmationDialog.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import { Box } from '@mui/system';
+
+export type ConfirmationDialogProps = {
```

Comment:
Use a union type with a basic type from `Dialog.tsx` to avoid redeclaring `onClose` and `onConfirm`. Might make sense to ditch content in favour of recycling children since strings can be react nodes.